core/src/main/resources/jenkins/telemetry/impl/AutoRefresh/description.jelly

```diff
@@ -0,0 +1,5 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core">
+    This trial collects information about the "Auto Refresh" feature to determine whether the feature is used enough to not remove it in a future release of Jenkins.
+    We collect the number of HTTP sessions that have the feature (typically controlled via a cookie) enabled or disabled.
```

Comment:
While this is technically accurate, the way it's written led me to think you're only talking about users that play w/ the feature.

I'd rather "collect the number of HTTP sessions and the number of HTTP sessions with the feature enabled/disabled".

Reply:
Yours is incorrect, as we don't know the total number of HTTP sessions as implemented.

The lower bound for the total is `Math.max(enabled, disabled)`, the upper bound is `enabled + disabled`.

Reply:
I can't decipher this. Can you describe real connections/users that are and aren't counted? And describe whether a user can be both using and not using this and how they're counted?

Reply:
We count everything, but if a user switches between enabled and disabled they're counted for both. Previously discussed in https://github.com/jenkinsci/jenkins/pull/3894#discussion_r256797535